android/cli/commands/_build.js

```diff
@@ -3689,7 +3689,17 @@ AndroidBuilder.prototype.buildAppProject = async function buildAppProject() {
 
 		// Set path to the app-bundle file that was built up above.
 		// Our "package.js" event hook will later copy it to the developer's chosen destination directory.
-		this.aabFile = path.join(this.buildDir, 'app', 'build', 'outputs', 'bundle', 'release', 'app.aab');
+		this.aabFile = path.join(this.buildDir, 'app', 'build', 'outputs', 'bundle', 'release', 'app-release.aab');
+	}
+
+	// Verify that we can find the above built file(s).
+	if (!await fs.exists(this.apkFile)) {
+		this.logger.error(`Failed to find built APK file: ${this.apkFile}`);
+		process.exit(1);
```

Comment:
Just an FYI - I am not fond of our use of exiting the process immediately in these functions like this. Ideally we simple return an error to the callback/throw an Error (if in an async function or sync function with no callback) - and let the top-level CLI do the exiting of the process with a failure exit code. We tend to have these calls sprinkled throughout CLI related code making re-use harder if we ever want to re-use in another context.

Reply:
I'll change it to throw an exception instead, which I think is fine in this case.

That said, I don't think we should throw an exception for say end-user configuration issues, because the logged stack trace would make it look like an issue on our end when it's not.